src/styles/index.scss

```diff
@@ -1,3 +1,99 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding-inline: 16px;
+
+  &__image {
+    width: 160px;
+    padding-top: 32px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+  }
+  span {
+    font-size: 10;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__button {
+    width: 160px;
+    height: 40px;
+    margin-top: 16px;
+    color: #fff;
+    background-color: #00acdc;
+    border-radius: 5px;
+    border: none;
+  }
+
+  &__button:hover {
+    color: #00acdc;
+    background-color: #fff;
+  }
+}
+
+.rating {
+  &__block {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    gap: 4px;
+
+    span {
```

Comment:
Avoid defining styles for base elements (like 'span') within component classes, as it can lead to unexpected results and is not a BEM best practice. Instead, define a class for this element and style it using that class.